src/include/pbs_ifl.h

```diff
@@ -486,13 +486,6 @@ enum batch_op {	SET, UNSET, INCR, DECR,
 	EQ, NE, GE, GT, LE, LT, DFLT
 };
 
-/* PBS supported authentication methods */
-enum pbs_auth_method {
-	AUTH_RESV_PORT,      /* Reserved port authentication */
-	AUTH_MUNGE,           /* MUNGE Authentication */
-	AUTH_GSS           /* GSS Authentication */
-};
-
```

Comment:
If we remove this and we also don't have any PBS_SUPPORTED_AUTH_METHODS currently, then how do we invoke the appropriate libauth_xxx.so api ?

Reply:
load_auth_lib make use of pbs_conf.auth_method to get .so name.